### PR TITLE
Add --signingKeyPath and --signingKeyAlias options to build command

### LINKED ISSUE #490: Support for providing signing key path & alias via command line flag in build command
**Is your feature request related to a problem? Please describe.**
I'm generating a non-production keystore to sign the APK with for testing purposes only. I don't want to put the non-production path and alias of this signing key into my TWA manifest because it isn't actually what's used to sign the production release of the APK, so it might confuse future developers. It would be great if Bubblewrap could use a keystore path and alias that aren't supplied in the TWA manifest to sign the APK during the build command.

**Describe the solution you'd like**
Accept `--signingKeyPath` and `--signingKeyAlias` command line flags for the build command, which take precedence during the build command over what's specified in the TWA manifest.

**Describe alternatives you've considered**
**Key path and alias provided via environment variables:** Neither of these are particularly sensitive data, so environment variables don't seem necessary, but they could work, too.
**Putting the keystore path and alias into the TWA manifest:** Given that this keystore is generated on the fly for testing purposes, it may confuse future developers to put it in the TWA manifest when the actual (production) signing process involves working with another team to get the APK signed & released to prod.

**Additional context**
Happy to implement this myself if that'd be helpful!

### PR DESCRIPTION
**Pull request by jdecked**
Fixes #490.

### CODE REVIEW

File: packages/cli/README.md
Comment (PEConn) [inline packages/cli/README.md:122]: I think potentially we're overcommunicating here. I can't think of an example where a command line parameter to a tool is overridden by the value in the config file - the parameter always takes precedence. Also, personally I'd take the "incompatible with `--skipSigning` as implied.

I'd be fine with just:

`--signingKeyPath`: path to keystore to use for signing the built APK and App Bundle.

And assume the user can figure out the other two parts.

**Review by PEConn — APPROVED** [review]
Code looks good - got a comment on the README docs, but that's more of a loose opinion. Let's see what Andre thinks.

File: packages/cli/README.md
Comment (andreban) [reply packages/cli/README.md:122]: What I understand is that when `--skipSigning` is used, `--signingKeyPath` and `--signingKeyAlias` are ignored. I think incompatible makes it look like command will not work if those are used together, which is not the case? 

I agree that the user can probably figure out implications of `--skipSigning` , but it doesn't hurt to be explicit about it.  I'd just say other parameters related to signing are ignored when this is used. I wouldn't add extra information on this on `--signingKeyPath` and only have it on `--skipSigning`.

**Review by andreban — APPROVED** [review]
LGTM, with nit on incompatible vs ignored.

**Comment by jdecked** [comment]
Could one of y'all please do a release once this is merged? Waiting for the release to merge an internal branch at Twitter. 🙏

**Comment by andreban** [comment]
> Could one of y'all please do a release once this is merged? Waiting for the release to merge an internal branch at Twitter. 🙏

I'm planning a release on Tue (I'm OOO today and Monday)

**Pull request merged**
